pycbc/inference/models/marginalized_gaussian_noise.py

```diff
@@ -325,15 +336,20 @@ def _loglr(self):
         for det in wfs:
             if det not in self.dets:
                 self.dets[det] = Detector(det)
-            fp, fc = self.dets[det].antenna_pattern(
-                                    params['ra'],
-                                    params['dec'],
-                                    params['polarization'],
-                                    params['tc'])
-            dt = self.dets[det].time_delay_from_earth_center(params['ra'],
-                                                             params['dec'],
-                                                             params['tc'])
+
+            if self.precalc_antenna_factors:
```

Comment:
How is `precalc_antenna_factors` set to True in a run? Is this an argument for the model? I couldn't find where it would be set.

Reply:
It's already handled by the marginalization parent class. The features were implemented first for other cases, and this just extends the use to the marginalized time model.

Reply:
Ok, but I was asking how you set that in the config file and how that's passed to the model initialization. I couldn't figure it out from looking at the code, although I didn't dig too deep.

Reply:
Ah, it's automatically populated if you do sky marginalization so it's the same set of settings really. These factors are calculated in any case when the sky grid is set up.